examples/slash_commands/main.go

```diff
@@ -2,503 +2,77 @@ package main
 
 import (
 	"flag"
-	"fmt"
 	"log"
 	"os"
 	"os/signal"
-	"strings"
-	"time"
 
 	"github.com/bwmarrin/discordgo"
 )
 
-// Bot parameters
 var (
 	GuildID        = flag.String("guild", "", "Test guild ID. If not passed - bot registers commands globally")
 	BotToken       = flag.String("token", "", "Bot access token")
-	RemoveCommands = flag.Bool("rmcmd", true, "Remove all commands after shutdowning or not")
+	RemoveCommands = flag.Bool("rmcmd", true, "Weather to remove all commands after shutdowning or not")
```

Comment:
Also speaking of performance and ratelimit issues, I think it would be better to set the default value to `false` here. That way each time you run the application you won't need to wait until it deletes the commands.

Otherwise, in addition to usage of `ApplicationCommandBulkOverwrite`, we can replace this parameter with `-clean`, which when enabled would use `ApplicationCommandCreate` & `ApplicationCommandDelete` instead of `ApplicationCommandBulkOverwrite`. That way it also might make sense to separate functionality into `createCommands` function instead of doing it directly in main.

Reply:
Am I correct to assume that if we pass empty list of commands to `ApplicationCommandBulkOverwrite` it will delete all of the commands?